packages/iov-keycontrol/src/userprofile.ts

```diff
@@ -85,27 +93,15 @@ export class UserProfile {
     return (await Random.getBytes(24)) as Xchacha20poly1305IetfNonce;
   }
 
-  private static labels(entries: ReadonlyArray<KeyringEntry>): ReadonlyArray<string | undefined> {
-    return entries.map(e => e.label.value) as ReadonlyArray<string | undefined>;
-  }
-
-  private static ids(entries: ReadonlyArray<KeyringEntry>): ReadonlyArray<KeyringEntryId> {
-    return entries.map(e => e.id);
-  }
-
   public readonly createdAt: ReadonlyDate;
   public readonly locked: ValueAndUpdates<boolean>;
-  public readonly entriesCount: ValueAndUpdates<number>;
-  public readonly entryLabels: ValueAndUpdates<ReadonlyArray<string | undefined>>;
-  public readonly entryIds: ValueAndUpdates<ReadonlyArray<KeyringEntryId>>;
+  public readonly wallets: ValueAndUpdates<ReadonlyArray<WalletInfo>>;
```

Comment:
:+1: yes this is what I was thinking of as I added entryIds.
I like the new name better as well